### ISSUE TITLE
Do not use inspect.getargspec() in twisted/test/test_twistd.py on Python 3

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8706]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)| @rodrigc reported|
|-|-|
|Trac ID|trac#8706|
|Type|enhancement|
|Created|2016-07-28 21:40:44Z|
|Branch|https://github.com/twisted/twisted/tree/8706-rodrigc-getargspec-py3|

inspect.getargspec() is deprecated

<details><summary>Searchable metadata</summary>

```
trac-id__8706 8706
type__enhancement enhancement
reporter__rodrigc rodrigc
priority__normal normal
milestone__Python_3_x Python-3.x
branch__8706_rodrigc_getargspec_py3 8706-rodrigc-getargspec-py3
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1469742044864963 1469742044864963
changetime__1470429412889694 1470429412889694
version__None None
owner__rodrigc rodrigc

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_1"></a>|@rodrigc commented|
|-|-|

See:
https://github.com/twisted/twisted/pull/441

**Comment by twisted-trac** [comment]
|[<img alt="Lukasa's avatar" src="https://avatars.githubusercontent.com/u/1382556?s=50" width="50" height="50">](https://github.com/Lukasa)<a name="note_2"></a>|@Lukasa set owner to @rodrigc|
|-|-|

LGTM, merge when ready.

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_3"></a>|GitHub set status to `closed`|
|-|-|

In changeset 3a9498400eedeabdc5b938ac55461744f7d9c562
```
#!CommitTicketReference repository="" revision="3a9498400eedeabdc5b938ac55461744f7d9c562"
Merge pull request #441 from twisted/8706-rodrigc-getargspec-py3

Author: rodrigc
Reviewer: Lukasa
Fixes: #8706

Do not use inspect.getargspec() in twisted/test/test_twistd.py on Python 3
```
